baip-0002.md

```diff
@@ -0,0 +1,62 @@
+    BAIP: 0002
+    Title: Reform of loopholes in feed price mechanism
+    Authors: cn-vote  bitshareschina@163.com
+    Status: Draft
+    Type: Consensus 
+    Created: 2019-11-5
+
+
+# Abstract
+This BAIP defines reforms to the current feed price mechanism vulnerability. The specific program is: The` feed price` is the highest between the `current price` and the ` two-day moving average price` .
+# Motivation
+After the failure of [BSIP42](https://github.com/bitshares/bsips/blob/master/bsip-0042.md), the current feed price mechanism has major loopholes and serious negatives, many cex exchanges use our vulnerability to maliciously short,which seriously damaged the ecological balance, which caused us to suffer many unnecessary losses and hindered the development of the entire ecology.
+When the vulnerability has been expanded to be intolerable, in an emergency, the passage and execution of [BSIP76](https://github.com/bitshares/bsips/blob/master/bsip-0076.md) has temporarily blocked the expansion of the vulnerability. However, the current feed price mechanism is still in urgent need of reform.
+# Rational
+"The feed price is the highest between the ` current price` and the `two-day moving average price` ".
+This BAIP does not conflict with the previous consensus on the feed price of all the communities. The feed provider continue to collect the feed price according to the original community consensus, and the community consensus on the protection of the black swan([BSIP58](https://github.com/bitshares/bsips/blob/master/bsip-0058.md)) and the minimum feed price is continued([BSIP76](https://github.com/bitshares/bsips/blob/master/bsip-0076.md)).
```

Comment:
> This BAIP does not conflict with the previous consensus on the feed price of all the communities.

It does conflict with historic/original consensus & past price feed changes have not achieve network consensus prior to implementation - actions not supported by all communities.

> The feed provider continue to collect the feed price according to the original community consensus

Cool, so back to feeding accurate price feeds immediately, right? 😂

That was the original consensus, remember? All debtors agreed to this when they opened their debt positions prior to BSIP76.

> black swan

This is old/invalid terminology from when an MPA could not recover from global settlement, now a global settlement event does not qualify as a 'black swan' as there are multiple exit conditions/routes.

> the minimum feed price is continued (BSIP76)

Why? Does your BAIP not fully address the CEX short attack concerns? You claim this is urgent, but then propose delaying implementation of the 'reformed' price feed scripts indefinitely? (The only exit clause for BSIP76 is BTS price raising above threshold price - circumstances which can be directly impeded by the "multiple malicious CEX").

---

[What alternative designs & related works were considered?](https://github.com/bitshares/baips/blob/master/baip-0001.md#what-belongs-in-a-baip)

[What important objections/concerns have been raised during discussions?](https://github.com/bitshares/baips/blob/master/baip-0001.md#what-belongs-in-a-baip)